R/utils_render_footnotes.R

```diff
@@ -552,10 +552,12 @@ apply_footnotes_to_output <- function(data,
   if (nrow(footnotes_tbl_data) > 0) {
 
     if ("stub" %in% footnotes_tbl_data$locname &&
-        "rowname" %in% colnames(body)) {
+        !is.na(dt_boxhead_get_var_stub(data))) {
```

Comment:
It feels like if `"stub" %in% footnotes_tbl_data$locname`, and `is.na(dt_boxhead_get_var_stub(data))`, that should be an error. In fact, that should probably be an error at the time the footnote is added.